crates/uv/tests/pip_list.rs

```diff
@@ -131,92 +118,47 @@ fn list_editable() -> Result<()> {
     Installed 4 packages in [TIME]
      + anyio==4.3.0
      + idna==3.6
-     + poetry-editable==0.1.0 (from file://[WORKSPACE_DIR]/scripts/packages/poetry_editable)
+     + poetry-editable==0.1.0 (from file://[WORKSPACE]/scripts/packages/poetry_editable)
      + sniffio==1.3.1
     "###
     );
 
-    // Account for difference length workspace dir
-    let prefix = if cfg!(windows) { "file:///" } else { "file://" };
-
-    // Origin of lengths used below:
-    // - |Editable project location| = 25
-    // - expected length = 48
-    // - expected length - |Editable project location| = 23
-    // - |`[WORKSPACE_DIR]/`| = 16
-    // - |`file://`| = 7, |`file:///`| = 8 (windows)
-
-    let workspace_len_difference = workspace_dir.as_str().len() + 23 - 16 - prefix.len();
-    let find_divider = "-".repeat(25 + workspace_len_difference);
-    let replace_divider = "-".repeat(48);
-
-    let find_header = format!(
-        "Editable project location{0}",
-        " ".repeat(workspace_len_difference)
-    );
-    let replace_header = format!("Editable project location{0}", " ".repeat(23));
-
-    let find_whitespace = " ".repeat(25 + workspace_len_difference);
-    let replace_whitespace = " ".repeat(48);
-
-    let search_workspace = workspace_dir_re.as_str().strip_prefix(prefix).unwrap();
-    let replace_workspace = "[WORKSPACE_DIR]/";
```

Comment:
This was just way too complicated and brittle for me. I dropped all of this in favor of some filters that do not preserve the aligned snapshot.